src/bundle/DependencyInjection/IbexaTestCoreExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\TestCore\DependencyInjection;
```

Comment:
AFAICS, according to composer.json, this should be
```suggestion
namespace Ibexa\Bundle\Test\Core\DependencyInjection;
```
`IbexaTestCoreBundle` suffers from the same issue.

(or the other way around, not sure what was the intention).

Reply:
Fixed namespaces and added test for basic Kernel compilation in b7318312.

Reply:
Fixed namespaces and added test for basic Kernel compilation in b7318312.